script/tar-my-folder/customize.py

```diff
@@ -11,7 +11,7 @@ def preprocess(i):
     input_dir = env.get("CM_TAR_INPUT_DIR", "")
     if input_dir == "":
         return {'return': 1, 'error': 'Please set CM_TAR_INPUT_DIR'}
-    output_dir = env.get("CM_TAR_OUTPUT_DIR", "")
+    output_dir = env.get("CM_MLPERF_INFERENCE_SUBMISSION_BASE_DIR", env.get("CM_TAR_OUTPUT_DIR", ""))
```

Comment:
We should pass this env from the submission generation script. `tar-my-folder` CM script should not know anything specific about submission generation.